### PR TITLE
Fix ulf download and update dependencies 

### PR DESCRIPTION
**Pull request by GwendalBroudin-Emoko**
Hi,

Thanks for your very useful tool! 

I had some download issues. After some testing, it seems that `Page.setDownloadBehavior` now require an absolute path. 

I also updated dependencies and remove some `waitForTimeout` that doesn't seem necessary anymore.

Gwendal

### CODE REVIEW

**Comment by NonPolynomialTim** [comment]
First off, a big thanks to all of the contributors for the great tool!  Second, thank you @GwendalBroudin-Emoko for this fix. 
 And finally, is there any way this PR could get merged soon?  Without the path.resolve() the download fails now, and I'd like to use this package in a docker build setup without needing to pull a special branch.  Thank you so much!

**Comment by GwendalBroudin-Emoko** [comment]
Hi @NonPolynomialTim, effectively this could be great for us as well. In the meantime, you can download the package on our registry https://www.npmjs.com/package/@emoko/unity-activate

**Comment by NonPolynomialTim** [comment]
Ah, thank you!  I've temporarily added your package to my Dockerfile instead

**Comment by StephenHodgson** [comment]
@GwendalBroudin-Emoko I'm actually still running into one of the timeouts still.

Is there any way to increate it?

```
Waiting for selector `#new_conversations_create_session_form #conversations_create_session_form_password` failed: Waiting failed: 2000ms exceeded
```

**Comment by GwendalBroudin-Emoko** [comment]
Hi @StephenHodgson,

You could pass a custom timeout in this function as argument to replace default value (2000ms) [waitForSelector function](https://github.com/E-moko/unity-activate/blob/82d7edf4f44fc89db401338dcefc5761c020b726/src/crawler.ts#L135).

But 2000ms is pretty long enough, this could be another issue.

**Comment by StephenHodgson** [comment]
When I updated it to 5000 it fixed my issue. Just a slow runner I suppose.

**Comment by StephenHodgson** [comment]
> You could pass a custom timeout in this function as argument to replace default value (2000ms)

This would be a nice feature

File: src/crawler.ts [hunk 1]
```diff
@@ -134,22 +134,23 @@ export abstract class Crawler {
...
     }
 
     async waitAndClick(selector: string): Promise<void> {
         logger.debug(`waitAndClick: ${selector}`);
-        await this.page.waitForTimeout(1000);
 
         await this.page.waitForSelector(selector, { timeout: 2000 });
-        await this.page.evaluate(s => document.querySelector(s).click(), selector);
+        
+        await this.page.evaluate(s =>  {
```
Comment (StephenHodgson) [inline src/crawler.ts:146]: got an error here when building. I think you need to define the type string
```suggestion
        await this.page.evaluate((s: string) =>  {
```

**Comment by StephenHodgson** [comment]
Proposed more changes for this PR here: https://github.com/E-moko/unity-activate/pull/2

Seems like there was a race condition with selecting the `Next` button and `Download` button.

**Comment by mob-sakai** [comment]
Sorry for my late reply.
Thank you for your cooperation! 👍

**Comment by mob-sakai** [comment]
@GwendalBroudin-Emoko @StephenHodgson
I cherry-picked your commits to `main` branch. 👍

**Pull request closed without merging**